lms/static/sass/discussion/views/_create-edit-post.scss

```diff
@@ -11,6 +11,11 @@
   padding: $baseline;
   max-width: 1180px;
 
+  .post-errors {
+    padding: 0 !important;
```

Comment:
These are the styles we're overriding:
![](
![image](https://cloud.githubusercontent.com/assets/407639/18138043/ddfb770c-6f78-11e6-9290-7d05e18b5714.png)) (from [here](https://github.com/edx/edx-platform/blob/bjacobel/discussion-alerts/lms/static/sass/course/courseware/_courseware.scss#L479)) so as you can see we'd have to get quite a bit more specific (like adding an additional three selectors or something) to override those styles naturally. I don't like `!important`s very much either, but I think we need them 😓  I could move them to the shame file, though.